foodnet/membership/models.py

```diff
@@ -63,39 +67,43 @@ def __str__(self):
 
 class Member(models.Model):
```

Comment:
So there are a few things I had to take into consideration:
* More than one department can have the same member.
* One member can have many users including membership owner. (a school or a houshold; AFAIK there is also a requirement for this to be linked to single address)
* The department admin can invite a user to a member and to the correct department (at least for now)
* In future membership owner will probably be able to invite users.

I agree it is a bit overcomplicated and I would prefer to simplify it, do you have any suggestions?